docs/src/components/Navigation/SideNavigation/sections.ts

```diff
@@ -0,0 +1,94 @@
+export const sections = [
```

Comment:
Consider adding links to the theme and utilities pages here. Utilities is all placeholder at at the moment, so we could add it later once it's useful, but theme already has the color swatch lists so could potentially already add some value (and I have already directed one person to it who was looking for it but couldn't find it)